src/components/organisms/ChatSidebar/components/VenueChat/VenueChat.tsx

```diff
@@ -17,6 +17,8 @@ export const VenueChat: React.FC = () => {
   return (
     <div className="venue-chat">
       <Chatbox
+        // poll is available for Venue Chat only (displayPoll = true)
+        displayPoll
```

Comment:
hasPoll

Reply:
I think `displayPoll` is fine here. Or alternatively `withPoll` is a more common form of this sort of boolean prop at the 'external API' side of things.

Reply:
Agreed